visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.tsx

```diff
@@ -376,7 +376,7 @@ export class CohortEditor extends React.PureComponent<ICohortEditorProps, ICohor
             filter.arg = Array.from(Array(meta.sortedCategoricalValues.length).keys());
         } else if (meta.treatAsCategorical) {
             filter.method = FilterMethods.includes;
-            filter.arg = meta.sortedCategoricalValues as any[];
+            filter.arg = [...meta.sortedCategoricalValues] as any[];
```

Comment:
something to look out for, js passes arrays by value, so mutating the copy mutates the original. [...x] makes a shallow copy